### PR TITLE
Feature: Added multi-line parsing via double quotes & Fix: Pipelines

### PR DESCRIPTION
**Pull request by VijoPlays**
- Added multi-line parsing
- Fixed current pipeline errors

### CODE REVIEW

File: tests/dotenv.net.Tests/DotEnv.Fluent.Tests.cs [hunk 1]
```diff
@@ -127,6 +128,25 @@ public void ConfigShouldLoadEnvWithQuotedValues()
...
+            .WithEnvFiles(MultiLinesEnvFileName)
+            .WithTrimValues()
+            .Load();
+
+        EnvReader.GetStringValue("DOUBLE_QUOTE")
+            .Should()
+            .Be("double");
+        EnvReader.GetStringValue("DOUBLE_QUOTE_MULTI_LINE")
+            .Should()
+            .Be("doubler");
```
Comment (bolorundurowb) [inline tests/dotenv.net.Tests/DotEnv.Fluent.Tests.cs:144]: I think this should be 

```suggestion
            .Be("dou\nbler");
```

File: tests/dotenv.net.Tests/DotEnv.Fluent.Tests.cs [hunk 2]
```diff
@@ -127,6 +128,25 @@ public void ConfigShouldLoadEnvWithQuotedValues()
...
+        EnvReader.GetStringValue("DOUBLE_QUOTE_EVEN_MORE_LINES")
+            .Should()
+            .Be("doublest");
```
Comment (bolorundurowb) [inline tests/dotenv.net.Tests/DotEnv.Fluent.Tests.cs:147]: I think this should be:

```suggestion
            .Be("dou\n\nb\n\nlest");
```

Comment (VijoPlays) [reply tests/dotenv.net.Tests/DotEnv.Fluent.Tests.cs]: Made the adjustment :)

File: src/dotenv.net/Parser.cs [hunk 3]
```diff
@@ -1,71 +1,89 @@
...
-                continue;
-
-            var key = row.Key(index);
-            var value = row.Value(index, shouldTrimValue);
-            validEntries.Add(new KeyValuePair<string, string>(key, value));
+            var rawEnvRow = rawEnvRows[i];
+
+            if(rawEnvRow.StartsWith("#")) continue;
+
+            if (rawEnvRow.Contains("=\""))
```
Comment (bolorundurowb) [inline src/dotenv.net/Parser.cs:23]: This line will fail if the env is 

```
DOUBLE_QUOTE_MULTI_LINE= "dou
bler"
```

I have merged the PR in but will  have to introduce a fix

**Pull request merged**